science/projects/Ghazaleh Mahmoudi/01-main-content/main_content_detection.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.                              content_words_number_list.append(len(str(content).split()))
</pre><p>using the advantage of wrapping all feature extractions in one function is to reuse variables such as str(content), casting it one time and using it multiple times seems to increase the efficiency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>